op-e2e/actions/proofs/l1_prague_fork_test.go

```diff
@@ -0,0 +1,164 @@
+package proofs
+
+import (
+	"fmt"
+	"testing"
+
+	batcherFlags "github.com/ethereum-optimism/optimism/op-batcher/flags"
+	"github.com/ethereum-optimism/optimism/op-chain-ops/genesis"
+	actionsHelpers "github.com/ethereum-optimism/optimism/op-e2e/actions/helpers"
+	"github.com/ethereum-optimism/optimism/op-e2e/actions/proofs/helpers"
+	"github.com/ethereum-optimism/optimism/op-program/client/claim"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/holiman/uint256"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPragueForkAfterGenesis(gt *testing.T) {
+	type testCase struct {
+		name         string
+		useSetCodeTx bool
+	}
+
+	testCases := []testCase{
+		{
+			name: "dynamicFeeTx", useSetCodeTx: false,
+		},
+		{
+			name: "setCodeTx", useSetCodeTx: true,
+		},
+	}
+
+	runL1PragueTest := func(gt *testing.T, testCfg *helpers.TestCfg[testCase]) {
+		t := actionsHelpers.NewDefaultTesting(gt)
+		env := helpers.NewL2FaultProofEnv(t, testCfg, helpers.NewTestParams(),
+			helpers.NewBatcherCfg(
+				func(c *actionsHelpers.BatcherCfg) {
+					c.DataAvailabilityType = batcherFlags.CalldataType
+				},
+			),
+			func(dp *genesis.DeployConfig) {
+				t := hexutil.Uint64(24) // Activate at second l1 block
+				dp.L1PragueTimeOffset = &t
```

Comment:
```suggestion
				dp.L1PragueTimeOffset = ptr(hexutil.Uint64(24)) // Activate at second l1 block
```
can do this after my pr is merged

Reply:
[e59430e](https://github.com/ethereum-optimism/optimism/pull/14006/commits/e59430e3acc984c5996fd71bdafdfdef2cf405c4)